lastpass-view.md

```diff
@@ -0,0 +1,24 @@
+# View Passwords That Have Been Shared With You
+This guide assumes that you have an [existing LastPass account](./lastpass-signup.md) and that some passwords have been shared with you by another user.
+
+## View Pending Shares
+Open the app to the Vault screen and look in the Pending Shares section.  Click on a pending entry in that list.  If no pending shares are visible, then you may not have any passwords shared with you yet.
+
+![View Pending Shares](./img/lastpass-view-1.jpg)
+
+## Accept the Share
+Accept the share, which will add the password to your account.
+
+![Accept the Share](./img/lastpass-view-2.jpg)
+
+## Look in Your Vault
+When LastPass returns to the vault screen, you should see the password that you just added.  Click on the "copy" button to copy the password to the clipboard.
+
+![Look in Your Vault](./img/lastpass-view-3.jpg)
+
+## Password is Copied
+You will see a notification that the password has been copied to the clipboard.  You can now go to the app and paste the password into the appropriate application.  Please note that the password will not be stored in your clipboard foever, it will usually be cleared after 2 minutes.
```

Comment:
```suggestion
You will see a notification that the password has been copied to the clipboard.  You can now go to the app and paste the password into the appropriate application.
- **NOTE**: password will not be stored in your clipboard forever, it will usually be cleared after 2 minutes.
```